### ISSUE TITLE
Log In screen form control missing textual label Chrome only

### ISSUE THREAD

**Issue by JohnHewittCriterion** [flexion/fs-intake-module#833]
Steps To Verify With JAWS:
 
In the Chrome browser, launch the website at the following URL: https://fs-intake-staging.app.cloud.gov/#/ 
Focus lands on the Password field. Press Shift + Tab to move backward one control. JAWS announces 'Edit'. Obviously this is the User Name field but it needs a textual label.
 
The field label is correctly announced by JAWS using Firefox and IE.
 
All form controls must have a meaningful textual label.

WCAG: 3.3 Input Assistance - 3.3.2 Labels or Instructions
EPERM-160

**Comment by dubharmonic** [comment]
The login screen no longer exists!
